tests/test_asciigraf.py

```diff
@@ -305,4 +374,29 @@ def test_missing_end_node_raises_missing_end_node_exception():
 Network String:
 1---
 Affected Edge:
- ---"""
+  --"""
+
+
+def test_bad_label_triggers_exception(caplog):
```

Comment:
Nice tests!

I like the descriptive errors.

Could you add a test for the "too few nodes" case?

Reply:
The test above already covers the "too few nodes" case